Patches.py

```diff
@@ -1291,6 +1291,17 @@ def set_entrance_updates(entrances: Iterable[Entrance]) -> None:
     rom.write_int16(0x00E1F3CA, 0x5036)
     rom.write_int16(0x00E1F3CC, 0x5036)
 
+    # Remove mask reaction text IDs from all actors not involved in the
+    # mask trading sequence. See z_face_reaction.c in decomp for original
+    # values or https://wiki.cloudmodding.com/oot/Code_(File)/NTSC_1.0#Mask_Reaction_Text.
+    # Text ID 0x0000 is the default if a mask is not being worn.
+    # Convenience hack to allow the player to keep the bunny hood on when interacting
+    # with actors that give items (rolling goron, lab dive, etc.)
+    if world.settings.auto_equip_masks:
```

Comment:
I don't think this will work as intended, since it's checking the value of a cosmetic setting at patch creation time.

Reply:
Thanks, you're right, oversight on my part

Reply:
This was moved to the cosmetic function with new logic to add in the original table if the cosmetic is disabled.